### ISSUE TITLE
Improve installation documentation

### ISSUE THREAD

**Issue by qx-775** [Volmarg/personal-management-system#65]
Hi,

I wanted to try this project and run it locally to test it. 
However, the docs could be improved.

I see mentions of Docker, but I could not find the docker container for this project, other than the docker-compose, and running composer inside some docker container, but that didn't contain the docker container for the personal-management-system. Does a docker container for this project exist? 

Also, it is unclear if only Linux is supported. I don't see anything about windows/mac. Is it not supported? Perhaps it's good to say that in the docs that only Linux is supported. 

On what port does the project run after installing? Perhaps it's also worth mentioning in the docs/readme.

Also, perhaps it would be nice to use the GitHub wiki with short URLs to installation, removal, security etc...

**Comment by Volmarg** [comment]
Hello,

There is a `docker-compose` which works with windows composer and under Linux (I've tested this in both cases).

- **Improving installation documentation**: this needs to wait as I got other priority for now,
   -  also I have a plan to create just `one-time` docker script which will trigger all the things needed to do with docker but again it has to wait,
 - **Installation**: this is also in plan (installer like WordPress has `step-by-step` via GUI)
 - **Support for windows**: partially (it generally works) but there is a problem with `file upload/management` which I'm aware of (someone reported it for me) - like above no time for now,
   - else other modules work fine on windows,
   - on Linux it works without issues (this is how i use it),
 - **Github Wiki** - not doing it, this is why I created a documentation,

So pretty much the installation improvement / bugs for windows - I'm aware of them all, but I need to prioritize things as I'm working on this on my own.

---

If You have an issue with installation - then let me know where / on which step.
